index.bzl

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Public API surface is re-exported here.
```

Comment:
could you use webpack/defs.bzl instead?

that follows the style guide https://docs.bazel.build/versions/main/skylark/deploying.html#rules

Reply:
I had trouble accessing the defs.bzl file. but I will dive deeper today and see why I can’t access.